pieforproviders/pieforproviders
# [PIE-2212] Signup form changes - API

## 💅🏼 What issue does this fix?



Adds `state` to the user model, adds survey questions to the user model.  Backend work for #2212 

## 🍂 Before You Submit


## 🛷 Deployment


## 🧵 Steps to set up locally

```
bundle exec rails db:migrate
```

## 🧳 Steps to test


- Send a registration request to the API using the state param.  It should successfully create a user.
- Send a registration request to the API without the state param.  It should return an error.
- Send a registration request to the API with the state param and survey question answers (one of "True", "Mostly True", "Mostly False", "False").  It should successfully create a user.
- Send a registration request to the API with the state param and a disallowed value for the survey question answers (anything but "True", "Mostly True", "Mostly False", or "False").  It should return an error.
